projects/adrv9009/src.mk

```diff
@@ -49,31 +49,31 @@ INCS :=	$(PROJECT)/src/app/app_config.h					\
 	$(PROJECT)/src/devices/adi_hal/adi_hal.h			\
 	$(PROJECT)/src/devices/adi_hal/common.h				\
 	$(PROJECT)/src/devices/adi_hal/parameters.h			\
-	$(PROJECT)/src/devices/talise/talise_agc.h			\
-	$(PROJECT)/src/devices/talise/talise_agc_types.h		\
-	$(PROJECT)/src/devices/talise/talise_arm.h			\
-	$(PROJECT)/src/devices/talise/talise_arm_macros.h		\
-	$(PROJECT)/src/devices/talise/talise_arm_types.h		\
-	$(PROJECT)/src/devices/talise/talise_cals.h			\
-	$(PROJECT)/src/devices/talise/talise_cals_types.h		\
-	$(PROJECT)/src/devices/talise/talise_error.h			\
-	$(PROJECT)/src/devices/talise/talise_error_types.h		\
-	$(PROJECT)/src/devices/talise/talise_gpio.h			\
-	$(PROJECT)/src/devices/talise/talise_gpio_types.h		\
-	$(PROJECT)/src/devices/talise/talise.h				\
-	$(PROJECT)/src/devices/talise/talise_hal.h			\
-	$(PROJECT)/src/devices/talise/talise_jesd204.h			\
-	$(PROJECT)/src/devices/talise/talise_jesd204_types.h		\
-	$(PROJECT)/src/devices/talise/talise_radioctrl.h		\
-	$(PROJECT)/src/devices/talise/talise_radioctrl_types.h		\
-	$(PROJECT)/src/devices/talise/talise_reg_addr_macros.h		\
-	$(PROJECT)/src/devices/talise/talise_rx.h			\
-	$(PROJECT)/src/devices/talise/talise_rx_types.h			\
-	$(PROJECT)/src/devices/talise/talise_tx.h			\
-	$(PROJECT)/src/devices/talise/talise_tx_types.h			\
-	$(PROJECT)/src/devices/talise/talise_types.h			\
-	$(PROJECT)/src/devices/talise/talise_user.h			\
-	$(PROJECT)/src/devices/talise/talise_version.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_agc.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_agc_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_arm.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_arm_macros.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_arm_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_cals.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_cals_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_error.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_error_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_gpio.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_gpio_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise.h				\
+	$(DRIVERS)/rf-transceiver/talise/talise_hal.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_jesd204.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_jesd204_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_radioctrl.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_radioctrl_types.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_reg_addr_macros.h		\
+	$(DRIVERS)/rf-transceiver/talise/talise_rx.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_rx_types.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_tx.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_tx_types.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_types.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_user.h			\
+	$(DRIVERS)/rf-transceiver/talise/talise_version.h			\
 	$(PROJECT)/src/firmware/talise_arm_binary.h			\
 	$(PROJECT)/src/firmware/talise_stream_binary.h			\
```

Comment:
The API is firmware dependent, so both should be kept together.

Reply:
Correct, it needs to be moved together